samples/model-builder/conftest.py

```diff
@@ -238,7 +238,7 @@ def mock_batch_predict_model(mock_model):
 
 @pytest.fixture
 def mock_upload_model():
-    with patch.object(aiplatform.models.Model, "upload") as mock:
+    with patch.object(aiplatform.Model, "upload") as mock:
         yield mock
```

Comment:
We can should return a mock `Model` object here
```py
@pytest.fixture
def mock_upload_model(mock_model):
    with patch.object(aiplatform.Model, "upload") as mock:
        mock.return_value = mock_model
        yield mock
```

Reply:
Thanks, done.